packages/core/lib/config-migration.js

```diff
@@ -8,20 +8,22 @@ const TruffleConfig = require("@truffle/config");
 const debugModule = require("debug");
 const debug = debugModule("core:config-migration");
 
+const CURRENT_CONFIG_VERSION = 1;
```

Comment:
Since we have a config version, should we add this output to `truffle version` ?

Reply:
I don't think this is worth doing. we don't have output for contract-schema version, e.g.